src/main/java/codeu/controller/ChatServlet.java

```diff
@@ -140,8 +141,12 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
 
     String messageContent = request.getParameter("message");
 
-    // this removes any HTML from the message content
-    String cleanedMessageContent = Jsoup.clean(messageContent, Whitelist.none());
+    OutputSettings settings = new OutputSettings();
+    settings.prettyPrint(false);
+
+
+    // Allows basic HTML
```

Comment:
Perhaps a little more detail can be added in this comment since it's difficult to know from a glance what Whitelist.basic() actually allows, maybe you can add the fact that unsafe tags like img and script are disallowed